hera_qm/utils.py

```diff
@@ -601,3 +601,44 @@ def strip_extension(path, return_ext=False):
         return (root, ext[1:])
     else:
         return os.path.splitext(path)[0]
+
+def read_bounds_text_file(list_file=None):
+    """Read in apriori flags
+
+    This method reads in a plaintext comma-separated file
```

Comment:
Calling it "comma-separated" is confusing, since it uses both commas and new-lines as delimiters. I'd be more explicit here.

It also occurs to me what we might want to use tomls (which support notions of nested arrays). I'm worried we're reinventing the wheel here.

Reply:
If we use a toml, this could be easily extended to channel numbers (and maybe even integration numbers / JDs / LSTs). Since a toml lets us define variable arrays, we could define one array as freq flags (using nested arrays). We could define another one for channel flags which could be a mixed nested array:

`channel_flags = [0, 1, [10, 20], 60]`

which would flag channels 0, 1, 10, 11, ... 19, 20, 60. 

We could even do both, freq_flags and channel_flags, with the caveat that the output of read_bounds will always be either channels or flags (your choice) and will require a frequency array to convert them if other type (or both) is provided.

This could definitely be extended to times in a way that supports unified I/O and it feels a lot more "pythonic" than a slightly janky text file. What do you think?